views/includes/underscore-templates.jade

```diff
@@ -36,14 +36,20 @@ script(type='text/html', id='useResultsTableTemplate').
     <% _.each(items.data, function(item, key, list) { %>
       <tr class="base--tr">
         <td class="base--td"><%= item.name %></td>
-        <td class="base--td"><%= item.score %></td>
+        <td class="base--td"><%= item.score.toFixed(3) %></td>
+        <td class="base--td thermometer"><img src="/thermometer?score=<%= item.score %>"</td>
       </tr>
   <% });%>
-  <% _.each(items.data, function(item, key, list) { %>
-  <% if (item.type_hierarchy && item.type_hierarchy.length > 0) { %>
-    <tr class="base--tr">
-      <td class="base--td" colspan="2">Type Hierarchy: <%= item.type_hierarchy %></td>
+  <% if (_.filter(items.data, function(item) { return item.type_hierarchy && item.type_hierarchy.length > 0; }).length > 0) { %>
```

Comment:
It might be worth putting `_.filter(items.data, function(item) { return item.type_hierarchy && item.type_hierarchy.length > 0; })` into a variable.. but not a blocking issue.